roles/common/tasks/main.yml

```diff
@@ -20,6 +20,33 @@
     retries: 5
   when: ansible_distribution == "Ubuntu"
 
+- name: Register with RHN
+  redhat_subscription:
+    state: present
+    username: "{{ redhat.user }}"
+    password: "{{ redhat.pass }}"
+  when: ansible_distribution == "RedHat"
+
+- name: Add OSP pool
+  command: subscription-manager attach --pool={{ item }}
+  when: ansible_distribution == "RedHat"
+  with_items:
+  - 8a85f98158880f8d01588bde0f7c3c43
+
+- name: add OS respositories
```

Comment:
Can we add a sleep, or someway to validate if the host is registered? I think it takes around a minute after you register before you can add pools or repos which kills the ursula run.